README.md

```diff
@@ -38,28 +17,22 @@ As a default the standard english `bert-base-uncased` model is used as a pretrai
 It is possible to use custom models for other languages. Below we describe how to do this for polish roberta model:
```

Comment:
Good idea - I modified the code and updated the readme. Now it can be used with either specifying model name from huggingface or path to the downloaded model.

Reply:
Great!